apps/dashboard/src/app/components/DashboardTypeBanner.tsx

```diff
@@ -0,0 +1,21 @@
+import { AnnouncementBanner } from "../../components/notices/AnnouncementBanner";
+
+export function TeamsUIBanner() {
+  return (
+    <AnnouncementBanner
+      label="Teams UI is still in Beta, Go to Old dashboard for fully functional dashboard"
+      href="/dashboard"
+      trackingLabel="old-dashboard"
+    />
+  );
+}
+
+export function TryTeamsUIBanner() {
+  return (
+    <AnnouncementBanner
+      label="Explore the new teams UI, Now in Beta"
```

Comment:
"Explore the new teams dashboard. Now in Beta."

Reply:
updated